### ISSUE TITLE
Javadoc all the things!

### ISSUE THREAD

**Issue by matys18** [matys18/Inda-project#26]
We should really start documenting our classes, to make it easier for us (and others) to understand what we've done. This does not have to happen overnight, but before the end of the project every class, method, interface etc. should be well documented.

@matys18 I can document the Model classes, the GameView and the Level\* classes. 

@Jarlius You should document your input controllers, the MenuView and ParentScreen classes. 👍 

So basically we just have to write javadocs for the code that we wrote 😄

**Comment by matys18** [comment]
The deadline for the project is this Wednesday, so we have to have everything documented by then ⚠️  Just a reminder that some of your classes such as MenuControl need some javadocs. It would be good to get that done before the övning. 👍  Also, this is not directly related, but there is an empty interface called InputReceiver, if it does not do anything it should be removed 💥

**Comment by Jarlius** [comment]
What is the purpose of {@inheritDoc} ? Should I use them on most if not all methods?

**Comment by Jarlius** [comment]
And the `InputReceiver` interface is what the `UserInput` sends its information to, be it `GameControl` or `MenuControl`. Sure, I could just let `UserInput` have a receiver of class `Object`, but I felt this was more elegant.

There's little reason to keep it though if it's unnecessary - if we keep it though, I'll give it a real documentation.

**Comment by matys18** [comment]
@Jarlius Oh ok. It's up to you, I didn't see that it was used like that :+1:

**Comment by Jarlius** [comment]
Oh I keep forgetting you can mention the issue in the commit message...
oh well, it's all documented now :sunny:
